src/main/java/tutoraid/logic/commands/AddStudentToLessonCommand.java

```diff
@@ -16,74 +18,111 @@
 
 
 /**
- * Adds a student to a lesson in TutorAid.
+ * Adds students to lessons in TutorAid.
  */
 public class AddStudentToLessonCommand extends AddCommand {
 
     public static final String COMMAND_FLAG = "-sl";
 
-    public static final String MESSAGE_USAGE = COMMAND_FLAG + ": Adds a student to a lesson "
-            + "by the student index number used in the last student listing "
-            + "and the lesson index used in the last lesson listing.\n"
-            + "Parameters: "
-            + PREFIX_STUDENT + "STUDENT_INDEX (must be a positive integer) "
-            + PREFIX_LESSON + "LESSON_INDEX (must be a positive integer) "
-            + "Example: " + COMMAND_FLAG
-            + " s/1"
-            + " l/2";
+    public static final String MESSAGE_USAGE = String.format("%1$s %2$s: Adds a student to a lesson. "
+                    + "\nParameters:"
+                    + "\n%3$sSTUDENT INDEX [MORE INDEXES] (must all be positive integers)"
+                    + "  %4$sLESSON INDEX [MORE INDEXES] (must be a positive integer)"
```

Comment:
Maybe can change `(must be a positive integer)` to `(must all be positive integers)` since this is the case for student index as well?